zpeclient.go

```diff
@@ -300,3 +303,39 @@ func (c *Cache) authorize(principal string, check AthenzAccessCheck) (bool, erro
 	}
 	return false, nil
 }
+
+func (c *Cache) checkUpdateTime(configmap string) (res bool, err error) {
```

Comment:
Can we move this entire logic to be done during configmap events? We can use the informer create and update event functions in order to process the configmap and set a timestamp variable which we can check directly in the authorize function. This will improve our performance as we do not need to all this parsing during the request flow.